test/cypress/support/index.js

```diff
@@ -31,3 +31,7 @@ Cypress.Commands.add('chart', () =>
         }
     }))
 );
+
+Cypress.Commands.add('openIndicators', () =>
+    cy.get('.highcharts-menu-item-btn').first().click()
```

Comment:
Instead of the first button, would it be better to get `.highcharts-indicators .highcharts-menu-item-btn`?